benchmarks/conftest.py

```diff
@@ -0,0 +1,152 @@
+from attr import dataclass
+import json
+import pytest
+
+from ariadne import QueryType, gql, make_executable_schema
+
+
+@dataclass
+class Person:
+    name: str
+    age: int
+
+
+@dataclass
+class User:
+    id: int
+    name: str
+    group: dict
+    avatar: list
+
+
+with open("benchmarks/simple.json") as f:
+    simple_data = json.load(f)
+
+with open("benchmarks/complex.json") as f:
+    complex_data = json.load(f)
```

Comment:
I'm wondering if it wouldn't be simpler to generate single `data.json` file that contains data for entire schema and then create two sets of data from it:

- `raw_data` which contains dicts
- `hydrated_data` which contains objects

Reply:
Now we have only one file data.json